### DISCUSSION TITLE
Nativewind 4 and Material Top Tabs - Tab alignment issues

### DISCUSSION THREAD

**Discussion by foureight84** [nativewind/nativewind#852] [Q&A]
I just added Nativewind 4.0.36 to my Expo 50 project that's currently using Material Top Tabs and I am seeing that the tabs alignment are off. Not sure how to fix this issue.

 

https://github.com/marklawlor/nativewind/assets/1146028/17c96b8a-1d93-40f7-9af1-01cc46061e2c

**Comment by foureight84** [comment]
> I just added Nativewind 4.0.36 to my Expo 50 project that's currently using Material Top Tabs and I am seeing that the tabs alignment are off. Not sure how to fix this issue.
> 
>  Recording.2024-03-16.051332.mp4

NVM.

```JAVASCRIPT
tabBarContentContainerStyle: {
          flex: 1,
          flexDirection: "row",
          justifyContent: "space-around",
        },
```
fixed the issue

**Comment by bhyoo99** [comment]
@foureight84 

<img width="486" alt="image" src="https://github.com/marklawlor/nativewind/assets/62757768/f97c0a99-fdd0-4c33-9fde-523e0c87ec15">

I don't think it's a good way to fix it. The pressable area is different.
